dev/t3c/run.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -o errexit
+trap '[ $? -eq 0 ] && exit 0 || echo "Error on line ${LINENO} of ${0}"; exit 1' EXIT
+
+
+cd "$TC/cache-config"
+
+# Build area may contain non-debug binaries
+make clean && make -j debug
+
+if [[ ! -f /usr/bin/t3c ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c/t3c /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-apply ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-apply/t3c-apply /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-check ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-check/t3c-check /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-check-refs ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-check-refs/t3c-check-refs /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-check-reload ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-check-reload/t3c-check-reload /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-diff ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-diff/t3c-diff /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-generate ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-generate/t3c-generate /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-preprocess ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-preprocess/t3c-preprocess /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-request ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-request/t3c-request /usr/bin/
+fi
+if [[ ! -f /usr/bin/t3c-update ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/t3c-update/t3c-update /usr/bin/
+fi
+if [[ ! -f /usr/bin/tm-health-client ]]; then
+	ln -s /root/go/src/github.com/apache/trafficcontrol/cache-config/tm-health-client/tm-health-client /usr/bin/
+fi
```

Comment:
Can this be a *for* loop?

Reply:
You know, I wanted to do that but initially I had thought that was going to be looping over a list of prohibitively long names, but I realize now I can make that a bit shorter.